### PR TITLE
Master pr

### PR DESCRIPTION
**Pull request by TheVeggydude**
V1.0.0 - Mean

### CODE REVIEW

File: docs/presentation/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Contributing
```
Comment (FlyingBird95) [inline docs/presentation/CONTRIBUTING.md]: Why this file, is it relevant for others?

File: docs/presentation/LICENSE [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Copyright (C) 2017 Hakim El Hattab, http://hakim.se, and reveal.js contributors
```
Comment (FlyingBird95) [inline docs/presentation/LICENSE]: Why this License?

File: docs/presentation/css/theme/league.css [hunk 3]
```diff
@@ -0,0 +1,270 @@
+/**
```
Comment (FlyingBird95) [inline docs/presentation/css/theme/league.css]: Maybe it's an idea to remove the themes that are not used

File: docs/presentation/plugin/markdown/example.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
+<!doctype html>
```
Comment (FlyingBird95) [inline docs/presentation/plugin/markdown/example.html]: Example file, so it can be removed

File: docs/presentation/plugin/markdown/example.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# Markdown Demo
```
Comment (FlyingBird95) [inline docs/presentation/plugin/markdown/example.md]: Also example file ==> also remove

**Review by FlyingBird95 — CHANGES_REQUESTED** [review]

File: docs/presentation/CONTRIBUTING.md (hunk 1 above)
Comment (InputUsername) [reply docs/presentation/CONTRIBUTING.md]: It's part of Reveal.js which @Qqwy used to make the presentation slides. The presentation files aren't really related to the rest of the project but I'm not sure which can be safely removed

File: docs/presentation/LICENSE (hunk 2 above)
Comment (InputUsername) [reply docs/presentation/LICENSE]: Same as the comment above

**Comment by Qqwy** [comment]
Ok! I'll take a minute to remove the not-so-important files from the presentation directory ;-)

**Review by FlyingBird95 — APPROVED** [review]

**Pull request merged**